### PR TITLE
add image picker with styles in CreateItem

### PR DESCRIPTION
**Pull request by yang6020**

### CODE REVIEW

File: js/screens/CreateItem/CreateItem.js [hunk 1]
```diff
@@ -4,132 +4,191 @@ import {
...
       SwitchValue: false,
+      photos: []
     };
   }
 
+  pickImage = (num) => {
+    let photoArray = [...this.state.photos]
+    ImagePicker.showImagePicker(options, (response) => {
+      if (response.didCancel) {
+      }
```
Comment (siddharthparmar7) [inline js/screens/CreateItem/CreateItem.js]: this should not be empty if statement.

File: js/screens/CreateItem/CreateItem.js [hunk 2]
```diff
@@ -4,132 +4,191 @@ import {
...
+      else if (response.error) {
+        console.log('ImagePicker Error: ', response.error);
```
Comment (siddharthparmar7) [inline js/screens/CreateItem/CreateItem.js]: no more console logs. throw an error or return it.

File: js/screens/CreateItem/styles.js [hunk 3]
```diff
@@ -1,11 +1,35 @@
 import { StyleSheet, Dimensions } from 'react-native';
 import { fonts, colors } from '../../config/styles';
 const { height, width } = Dimensions.get('window');
+const heightimg = height / 4 - 7
+const widthimg = width / 3 - 7
```
Comment (siddharthparmar7) [inline js/screens/CreateItem/styles.js]: what? why are your height and width are negative numbers?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes. Thank you.

File: js/screens/CreateItem/styles.js [hunk 4]
```diff
@@ -1,11 +1,36 @@
 import { StyleSheet, Dimensions } from 'react-native';
 import { fonts, colors } from '../../config/styles';
 const { height, width } = Dimensions.get('window');
+const heightimg = height / 4.27;
+const widthimg = width / 3.2;
```
Comment (siddharthparmar7) [inline js/screens/CreateItem/styles.js]: Please change this as asked earlier.

File: js/screens/CreateItem/styles.js [hunk 5]
```diff
@@ -1,11 +1,36 @@
...
   container: {
     backgroundColor: 'white',
-    paddingHorizontal: 10,
+    flex: 1,
+    paddingTop: 10,
+    padding: 10,
+  },
+  selectedImage: {
+    width: widthimg,
+    height: heightimg,
```
Comment (siddharthparmar7) [inline js/screens/CreateItem/styles.js]: if you give flex property to an image it should work fine without giving it height and width. Make all the necessary changes in this file.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes that I asked for earlier. Thank you.

File: js/screens/CreateItem/styles.js (hunk 5 above)
Comment (yang6020) [reply js/screens/CreateItem/styles.js]: hi sid, the Image tag should have height and width or it won't display an image

File: js/screens/CreateItem/styles.js [hunk 6]
```diff
@@ -5,8 +5,50 @@ const { height, width } = Dimensions.get('window');
...
+    alignSelf: 'center',
   },
+  defaultImageContainer: {
+    backgroundColor: 'white',
+    alignItems: 'center',
+    justifyContent: 'center',
+    alignSelf: 'center',
+    borderWidth: 1,
+    borderColor: colors.darkBlue,
+    width: '100%',
```
Comment (siddharthparmar7) [inline js/screens/CreateItem/styles.js:41]: Again? change it in your next pull request.

**Review by siddharthparmar7 — APPROVED** [review]
Make the required changes in the next pull request.

**Pull request merged**